src/_data/keywords.yml

```
@@ -0,0 +1,207 @@
+  type: bit
+- term: 'factory'
+  link: /language/constructors#factory-constructors
+  type: bit
+- term: 'false'
+  link: /language/built-in-types#booleans
+  type: reserved
+- term: 'final&nbsp;(var)'
+  link: /language/variables#final-and-const
+  type: reserved
+- term: 'final&nbsp;(class)'
+  link: /language/class-modifiers#final
+  type: bit
+- term: 'finally'
+  link: /language/error-handling#finally
+  type: reserved
+- term: 'for'
+  link: /language/loops#for-loops
+  type: reserved
+- term: 'Function'
+  link: /language/functions
+  type: bit
+- term: 'get'
+  link: /language/methods#getters-and-setters
+  type: bit
+- term: 'hide'
+  link: /language/libraries#importing-only-part-of-a-library
+  type: reserved
+- term: 'if'
+  link: /language/branches#if
+  type: reserved
+- term: 'implements'
+  link: /language/classes#implicit-interfaces
+  type: bit
+- term: 'import'
+  link: /language/libraries#using-libraries
+  type: bit
+- term: 'in'
+  link: /language/loops#for-loops
+  type: reserved
+- term: 'interface'
+  link: /language/class-modifiers#interface
+  type: bit
+- term: 'is'
+  link: /language/operators#type-test-operators
+  type: reserved
+- term: 'late'
+  link: /language/variables#late-variables
+  type: bit
+- term: 'library'
+  link: /language/libraries
+  type: bit
+- term: 'mixin'
+  link: /language/mixins
+  type: bit
+- term: 'new'
+  link: /language/classes#using-constructors
+  type: reserved
+- term: 'null'
+  link: /language/variables#default-value
+  type: reserved
+- term: 'of'
+  link: /guides/libraries/create-packages#organizing-a-package
+  type: context
+- term: 'on'
+  link: /language/error-handling#catch
+  type: context
+- term: 'operator'
+  link: /language/methods#operators
+  type: bit
+- term: 'part'
+  link: /guides/libraries/create-packages#organizing-a-package
+  type: bit
+- term: 'required'
+  link: /language/functions#named-parameters
+  type: bit
+- term: 'rethrow'
+  link: /language/error-handling#catch
+  type: reserved
+- term: 'return'
+  link: /language/functions#return-values
+  type: reserved
+- term: 'sealed'
+  link: /language/class-modifiers#sealed
+  type: bit
+- term: 'set'
+  link: /language/methods#getters-and-setters
+  type: bit
+- term: 'show'
+  link: /language/libraries#importing-only-part-of-a-library
+  type: context
+- term: 'static'
+  link: /language/classes#class-variables-and-methods
+  type: bit
+- term: 'super'
+  link: /language/extend
+  type: reserved
+- term: 'switch'
+  link: /language/branches#switch
+  type: reserved
+- term: 'sync'
+  link: /language/functions#generators
+  type: context
+- term: 'this'
+  link: /language/constructors
+  type: reserved
+- term: 'throw'
+  link: /language/error-handling#throw
+  type: reserved
+- term: 'true'
+  link: /language/built-in-types#booleans
+  type: reserved
+- term: 'try'
+  link: /language/error-handling#catch
+  type: reserved
+- term: 'type'
+  link: /language/extension-types
+  type: reserved
```

This one is really tricky. I couldn't convince the language team to make it a built-in identifier, so it isn't. But it is just like a built-in identifier in that it cannot be the name of "some" kinds of declarations, except that it is _only_ forbidden for an `extension` declaration to have the name `type`. That's again because it is confusing if `extension type ...` can be an `extension` whose name is 'type' rather than an `extension type`. So let's call it a built-in identifier. It's _almost_ true.
```suggestion
  type: bit
```